### DISCUSSION TITLE
Example of CSS Framework Import

### DISCUSSION THREAD

**Discussion by jpriollaud** [ElMassimo/vite_ruby#271] [Q&A] [answered]
Does anyone have a really simple example of a CSS npm framework config and import?

<details>
<summary>package.json</summary>

```json
"materialize-css": "^1.0.0-rc.2"
```

</details>

With `@use`, it does not find the `.scss` file in `node_modules`

```scsss
@use 'materialize-css/sass/materialize.scss'
```
```
[plugin:vite:css] [sass] Can't find stylesheet to import.
```

My scss imports work fine for my own custom styles, but not for imported style modules.

**Comment by jpriollaud** [comment, marked as answer]
Solution: 
`import 'materialize-css/sass/materialize.scss'`.

**Reply by jpriollaud** [reply]
for folks migrating to newer frameworks, this statement from the guide is true "_Vite provides [great support for CSS](https://vitejs.dev/guide/features.html#css), supporting PostCSS out of the box..." but if you try to setup Tailwind beware of **legacy configurations** in your project that will trip vite:

`.postcssrc.yml
`

**Reply by jpriollaud** [reply]
@ElMassimo a mention of this in the Migration guide would help quite a few.

**Reply by ElMassimo** [reply]
Hi there!

Different users will run into different problems when migrating.

Trying to add every possible scenario will increase the length of the docs, making it less likely that people will read the entire thing.

That's the reason I encourage people to share their experience using these _Discussions_ 😃 

Your post is indexed in search engines, and will already help users that are googling for this particular problem.
